addons/l10n_cz/models/account_move.py

```diff
@@ -5,4 +5,43 @@
 class AccountMove(models.Model):
     _inherit = 'account.move'
 
-    taxable_supply_date = fields.Date()
+    taxable_supply_date = fields.Date(copy=False)
+    reverse_charge = fields.Boolean(help='Received or realized taxable supplies in the domestic reverse charge regime')
+    # If a reverse charge transaction, user has to choose a reverse charge supply code.
+    supplies_code_reverse_charge = fields.Selection(
```

Comment:
no default ? Do we let the user choose, what about if he doesn't select anything ?